files/kule.lazy/update.json

```diff
@@ -0,0 +1,8 @@
+{
+	"packageManager": "github",
+	"name": "kule.lazy",
+	"repo": "keicheng/kule.lazy",
+	"files": {
+		"include": ["kule-lazy-full.min.css", "kule-effects.min.css", "kule-animates.min.css", "kule-reset.min.css", "kule-grid.min.css"]
```

Comment:
These files are not directly located under the root of the project's repository, may need to set `basePath` for it.

Ref:
https://github.com/jsdelivr/libgrabber#add-updatejson-schema